### ISSUE TITLE
Weight pretrain model

### ISSUE THREAD

**Issue by ntdat-8324** [myaxxxxx/onebit-st#2]
Hello, it's me again. I apologize for bothering you once more. I have dealt with my supervisor for the course, and we agreed that there is no need to train the model one more time, but just to execute the results on the test set as in your paper. Luckily, I have found the test set, but I can't proceed without the pre-trained model weights. I assure you that I will only use it for learning and research purposes. Would you be able to share it with me? I would greatly appreciate it. Thank you for your contributions.

**Comment by myaxxxxx** [comment]
Apologies for the delayed response. The experiment was completed about a year and a half ago, and at that time, the data and model were stored together. However, after the server was updated, the data and the model were deleted. I’m truly sorry that I’m unable to provide the pretrained model.
